src/index.js

```diff
@@ -148,11 +148,16 @@ IdTokenVerifier.prototype.verify = function(token, nonce, cb) {
  * Verifies that the `exp` and `nbf` claims are valid in the current moment.
  *
  * @method verifyExpAndNbf
+ * @deprecated Please use verify() instead
  * @param {String} exp value of `exp` claim
  * @param {String} nbf value of `nbf` claim
  * @return {boolean} if token is valid according to `exp` and `nbf`
  */
 IdTokenVerifier.prototype.verifyExpAndNbf = function(exp, nbf) {
+  console.warn(
```

Comment:
is this a common practice? On java, for example, just adding the `@deprecated` will suffice

Reply:
I've seen it on other libraries. Gives a bit more visibility if they do happen to call it.